modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/http/routes/DelegatedStakesRoutes.scala

```diff
@@ -200,5 +345,17 @@ final case class DelegatedStakesRoutes[F[_]: Async: Hasher](
           Ok(getLastReference(address, info))
         case None => ServiceUnavailable()
       }
+
+    case GET -> Root / "rewards-info" =>
+      snapshotStorage.head.flatMap {
+        case Some((snapshot, info)) =>
+          getRewardsInfo(snapshot, info)
+            .flatMap(Ok(_))
+            .handleErrorWith { error =>
+              logger.error(error)(s"Error getting rewards info: ${error.getMessage}") >>
```

Comment:
Maybe over-reaching here, but it'd be nice to track failures like this in a metric, so we don't have to scan logs.